app/views/api/v1/targets/show.json.jbuilder

```diff
@@ -0,0 +1,3 @@
+json.target do
+  json.extract! @target, :id, :topic_id, :title, :radius, :latitude, :longitude
```

Comment:
maybe you could move this to a 'info' partial, because in the future maybe you will make an ```index``` view where you will have to show user's targets info. WDYT?

Reply:
Thanks i already implement it!